### DISCUSSION TITLE
LSM6DSO

### DISCUSSION THREAD

**Discussion by GiuseppeTorre** [stm32duino/Arduino_Core_STM32#2298] [Q&A] [answered]
Hi, In the code for the LSM6DSO , there is accelerometer[0], accelerometer[1], accelerometer[2], gyroscope[0], gyroscope[1], gyroscope[2]. They are the x,y,z axis? Which is which?

**Comment by cparata** [comment, marked as answer]
Hello @GiuseppeTorre ,
exactly, the accelerometer and the gyroscope are vector quantities and accelerometer[0], accelerometer[1], accelerometer[2] represent respectively the x, y and z components for the acceleration and gyroscope[0], gyroscope[1], gyroscope[2] represent respectively the x, y and z components for the angular rate.
Best Regards,
Carlo

**Reply by GiuseppeTorre** [reply]
Thanks!
